app/assets/javascripts/application.js

```diff
@@ -68,8 +68,11 @@ $(document).on('railsAutocomplete.select', '#fake_searched_id', function(){
   $(this).parents('form').submit();
 });
 
-$(document).ready(function() {
+$(document).on('click', 'input.btn-primary:submit', function(){
```

Comment:
No it's redundant, I actually thought I got rid of it...

‫ב-27 בינו׳ 2016, בשעה 15:23, ‏‏Oren Kanner ‏<notifications@github.com> כתב/ה:‬

> In app/assets/javascripts/application.js:
> 
> > @@ -68,8 +68,11 @@ $(document).on('railsAutocomplete.select', '#fake_searched_id', function(){
> >    $(this).parents('form').submit();
> >  });
> >  
> > -$(document).ready(function() {
> > +$(document).on('click', 'input.btn-primary:submit', function(){
> Is this still necessary even after you figured out rebuilding the form?
> 
> —
> Reply to this email directly or view it on GitHub.
>